zap/src/main/java/org/zaproxy/zap/extension/api/PhpAPIGenerator.java

```diff
@@ -96,23 +114,23 @@ private void generatePhpElement(ApiElement element, String component, String typ
 
         try {
             String desc = getMessages().getString(descTag);
-            out.write("\t/**\n");
-            out.write("\t * " + desc + "\n");
+            out.write("/**\n");
+            out.write(" * " + addLinebreaks(desc, 100) + "\n");
```

Comment:
We need to split lines because of PHP standard requires 120 as max length. 100 is a best effort because of indentation.

Reply:
Is that standard enforced somehow? It would be better to validate and auto-format that in the target repo rather than do tweaks in the generator.

Reply:
Fixed.